### PR TITLE
README changes

### PR DESCRIPTION
**Pull request by c0dzilla**
Please discuss any issues on the thread before merge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,51 +1,39 @@
 # LimeLight
-> A disaster management ecosystem from the future
+> Disaster relief ecosystem to locate victims and coordinate operations
```
Comment (kritisingh1) [inline README.md]: IMO subheading can be kept short to just `Disaster relief ecosystem` or `Disaster management ecosystem` just to retain people's interest till the end of README. We're anyway describing what the app does in the following paragraph.

Comment (c0dzilla) [reply README.md]: I'm not sure whether `disaster relief ecosystem` or `disaster management system` gives enough context. What does one make of it? It sounds generic to me.

Comment (c0dzilla) [reply README.md]: But yes, on the other hand, sub-heading can be kept short as well so I'm really not sure.

File: README.md [hunk 2]
```diff
@@ -1,51 +1,39 @@
...
 
-A user using this application will be able to perform the following tasks:-
+An individual using this application will be able to perform the following tasks:-
 
-* View which zones are assigned to which users in the Map Overlay Mode
+* View the number of rescuers and medics present in a zone
 * Assign himself to a zone and move out to help those in need
 * Dissociate himself from a zone when most probably everybody in the zone has been saved
 * Request reinforcements from the pool of users who are currently un-assigned
-* Assign a task to a user or group of users ( Would require admin privileges to prevent abuse )
```
Comment (alphadose) [inline README.md]: I think this is necessary. We shouldn't leave every decision to individual members. It is entirely possible the rescuers and categorized into teams such as:-
Team Tango - 5 rescuers, 3 medics
Team Charlie - 4 rescuers, 5 medics
One should be able to give commands to a team as a whole

File: README.md [hunk 3]
```diff
@@ -1,51 +1,39 @@
...
+These locations would be rendered in a Google Map Interface in the rescuers' phones. A cluster of locations would comprise a rescue zone. By clicking on a marker in the map interface, information such as the number of devices in the cluster shall be visible. Now rescue operations can be carried out on a zonal basis. In case, there is no internet connectivity at a disaster site, the victims would be located via their phone's **bluetooth**. The range of a bluetooth is approximately 100 metres. Whenever a rescue worker steps within 100 metre radius of a victim, he would be notified of the victim's presence.
 
-* Scout - One who surveys the area and reports the damage and status of victims
-* Medic - One who is responsible for providing first-aid treatment to the victims
-* Lifter - One who removes wreckage and pulls out victims from underneath
+The second step is carrying out the rescue operations in the most optimal manner. There are mainly two kinds of people involved in the relief effort:
 
-There are only 3 roles in the initial draft but the infrastructure of the application would be such that it would allow defining any number of custom roles as per the requirements.
```
Comment (alphadose) [inline README.md]: The role system should be extensible. By hardcoding those roles we are restricting the ecosystem. Hence this line is necessary

File: README.md (hunk 1 above)
Comment (c0dzilla) [reply README.md]: @kritisingh1 can you please make the change on this branch removing the part after ecosystem?

**Comment by kritisingh1** [comment]
I think this is fine. Have a look in case anything else is needed @alphadose @c0dzilla and make a PR. I am merging this for now.

**Pull request merged**